### ISSUE TITLE
Fallen Empires expansion for games that are abandoned and unable to find replacement

### ISSUE THREAD

**Issue by yamac-kurtulus** [AsyncTI4/TI4_map_generator_bot#1526]
Hi,
I am writing this after one of our players abandoned a TIGL game in such a bad position that we'd never find a replacement for. We thought of eliminating them and continue the game, but it would take time and interfered with most players' plans.

I knew of [this mod on reddit](https://www.reddit.com/r/twilightimperium/comments/orvt4j/fallen_empires_miniexpansion_for_irl_games_with/), and tried it once in an IRL game after a player quitted a game, and it worked fairly well. It was fitting to the epic and we all felt like some catastrophe really happened in that part of the universe. 

TL;DR: what it does is that it enforces players to spend some sort of resource to acquiring quitting player's slice. It does have some random factor in it, so some cards are easier and some are harder to deal with, and each calamity has preferred factions.

I believe this would be a cool alternative to trying to scrunch players to hopeless situations and then lose all hope and end the game. But somewhat a large feature. What do you think?

**Comment by jamesrudolph** [comment]
[Homebrew HUB Thread](https://discord.com/channels/743629929484386395/1205399524550647910)
